manifests/bran.yml

```diff
@@ -0,0 +1,173 @@
+---
+name: perf-test
```

Comment:
❓ what is bran?

Reply:
@ameowlia it's Mona's favorite kind of muffin, and the arbitrary name we chose for our manifest. We could probably stand to change this.

Reply:
😹  yeah, please change. Also the correct favorite type of muffin is banana nut muffin 🍌 .

Reply:
Gotta go with @monamohebbi here, Bran muffins are way better than banana nut muffins.

As part of the cleanup phase we want to delete all the other manifests and just leave one modern bosh 2.0 manifest.

Reply:
Cool. I would argue that a muffin without any chocolate in it cannot be the best muffin...